src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -74,7 +74,11 @@ impl EventProcessor {
 
         // Handle IME requests.
         while let Ok(request) = self.ime_receiver.try_recv() {
-            let ime = window_target.ime.get_mut();
+            let ime = match window_target.ime.as_mut() {
+                Some(ime) => ime,
+                None => continue,
```

Comment:
Nit: I think `break` would be more appropriate here.

Reply:
We should drain until the error to not increase the queue over time.